### ISSUE TITLE
Bug: Boards not make the change without Ctrl+R

### ISSUE THREAD

**Issue by ikkinophoenix** [mattermost/focalboard#1769]
## Steps to reproduce the behavior

1. Go to Boards
2. Select  a current table
3. Move a card to another column or make a modification on a card
4. See error : the refresh is not automatic, i have to do a "Ctrl+R" to see the change

## Expected behavior

I have to do the "Ctrl + R" manipulation to refresh the page with each modification on a table (new cards or update of existing cards, moving a card to another column).
In short, the refresh and the taking into account of the modification is not done automatically without doing Ctrl + R

Note that in the first Beta version of FocalBoards under Mattermost 4.xx it works very well but not since version 6.xx

## Edition and Platform
 - Edition: Mattermost Server / Focalboard
 - Version: 6.0.1
 - Browser and OS: any

**Comment by chenilim** [comment]
Hi @ikkinophoenix, thanks for reporting this! There is a known refresh issue that was fixed in [v6.0.2](https://mattermost.com/blog/mattermost-security-updates-6-0-2-5-39-1-5-38-3-and-5-37-3-extended-support-release/). You can also upgrade Boards to v0.9.4 in the Marketplace to apply the fix:
1. Open the top-left menu in Mattermost, and select "Marketplace"
2. Search for "Boards", and select "Upgrade" (should be v0.9.4 or above)
3. Refresh your browser after the upgrade (Cmd+Shift+R)

Does that resolve the issue for you?

**Comment by ikkinophoenix** [comment]
Hi

It's ok now !!

I solve it with update the settings on file config.json
>> setting is "EnableReliableWebsockets": true

And it works !

Thanks

Regards


Jerome

De : Chen-I Lim ***@***.***>
Envoyé : lundi 8 novembre 2021 17:24
À : mattermost/focalboard ***@***.***>
Cc : Jérôme BOLOTRA ***@***.***>; Mention ***@***.***>
Objet : Re: [mattermost/focalboard] Bug: Boards not make the change without Ctrl+R (Issue #1769)


Hi @ikkinophoenix<https://m365.eu.vadesecure.com/safeproxy/v4?f=d_ShOw5idoNpKZx-HjyrcOWduC1LNbsgAhwvlKCw0nasSsQy4T2rOpif9PnFhMM9&i=Gtz10J-2iaJijpjkOnJp3kgIdp6l81pUV98MpgvHQ3aDNn12TR5qInmwFNiEmEoMn-tin4n5nx5fuqDzijLpzg&k=JlO6&r=vvakwSoQhd4xvqM7PNA_3ERqTS8mGqoJYbqtAC5vkqa-xVHA3BOp1ai6I3aNfRm8&s=ed609373ae70efee755c3a3196df03570d8ed2c170737805f86acd5a4bbe6851&u=https%3A%2F%2Fgithub.com%2Fikkinophoenix>, thanks for reporting this! There is a known refresh issue that was fixed in v6.0.2<https://m365.eu.vadesecure.com/safeproxy/v4?f=c5u6nKLUncFlvPraji_R-1tXXVPXnc44eglFuI-KRkDYasDsba_8Uk2Coar4FW9K&i=wfqQwU6avkslb8B6nMtbzyG8BEL_do-VXmrvOvA14XWsaqacEs-OLupITnj3pRyVh_dOh5U3KSihXxi7Gwox_g&k=WVBP&r=MfTki7ZSyc3lhGA6qnahnGRkroHnCHZtl9MmWEKW4Iw67QqBCfLDP5IarH755zdr&s=5eade7037ad59249583b5877bb8a194ef10dba13caacbc9303bc203c6b4ae19b&u=https%3A%2F%2Fmattermost.com%2Fblog%2Fmattermost-security-updates-6-0-2-5-39-1-5-38-3-and-5-37-3-extended-support-release%2F>. You can also upgrade Boards to v0.9.4 in the Marketplace to apply the fix:

  1.  Open the top-left menu in Mattermost, and select "Marketplace"
  2.  Search for "Boards", and select "Upgrade" (should be v0.9.4 or above)
  3.  Refresh your browser after the upgrade (Cmd+Shift+R)

Does that resolve the issue for you?

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://m365.eu.vadesecure.com/safeproxy/v4?f=GVbERn44wRSO3DYjmRcXHs6unsouU77n0pFSoJRE8O4ie5n5mfQik4ZdXjp2oLt8&i=o73vy0uVyQB3J7z4wbnkLVU-2qpWGkHbUsrGyr3cxly0sjKdvMOstS-BmQvKLuvq2zVApt5aqjFfZj-c5ZKb_A&k=2cmZ&r=wL-Shcv8ePgK8nwH8SuBFrW9Zwcv2EAoNOTgDOkXI4QmA-YjOhrI80mRg5yhuHD1&s=05c306805a58d126f3ae422afa7f55536fa063d8cffdcb7423262089a7505d85&u=https%3A%2F%2Fgithub.com%2Fmattermost%2Ffocalboard%2Fissues%2F1769%23issuecomment-963324400>, or unsubscribe<https://m365.eu.vadesecure.com/safeproxy/v4?f=qkCCxJyyvd0nBllluTuUo9rKVXYcOTLswTWl_v2mjxQXQfCB2Aoc0p1EBH14opFr&i=tYt9pafEN2t79ICEbz75tcRy0YlOwA7ye1gvA1ztSCDHlqtvArxwjqvzzWcX7izJv8mo64Sv4Zg7sUDsFobeGA&k=3Y5L&r=5UtAbX26-0PDogHadZrTMJejg68qdh0asc3nQdmRRYT5LEQYV8IFgVB3SvLiakug&s=95e89c17d265d8544d977db7f2ed69eb3e48c3cfc558f4526cb25b601dd6c73d&u=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWMOJYZ2BQTMWENTSQ4QDO3UK72RBANCNFSM5HR7WEHQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://m365.eu.vadesecure.com/safeproxy/v4?f=oJ2IM92cTOz3p3Xm8UsEdYaVENTggeOMeVi5nFUItG_3CA8AnMxHqdnn7aPPSgwt&i=qVl2FDDKDHOkquSJ20SMa-hiczAsO6kmblB0rJN1a_AvK3FBvrFBhA4irXFSwtwyPOozHC00KLs-ShqA2RbTRw&k=nWNc&r=5uXQ2_jKQaZEvvShd0OaFEV7cknZWBFHXkQRdodQVQ175wTlvyy0sOzzn5JmOS0S&s=de454b3d5adb2a9f8055b8a34a67b32f17711c1c7a78608ccf076663b77fe6a5&u=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675> or Android<https://m365.eu.vadesecure.com/safeproxy/v4?f=pV-RJDt8-wXqQO7O0z_3GHoXdVhI0ryFp03CL5AN2Mamvhq-ENGghVNkPgtL2ArO&i=boiUVQ3zCbVJ97o-wPxDaKfw_VwwB6y5iMnGG8C7xlmJI9Z3UZSPcqaRmXUYWYeBS03LnnrXDdzF9nJ28aj0KA&k=Hft0&r=-Fhn4PibbVPmuIlqofEJw6kmewYyBs-JVrVMRjDPGLlFtYYCmrtuyksrUB_tub0a&s=4dc1a64e5c5309cbe0b53d4d43155bc842490c2461d47991de30cee13846a01f&u=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub>.

**Comment by chenilim** [comment]
Thanks @ikkinophoenix!
